src/frontend/planner_test/tests/testdata/tpch_variant.yaml

```diff
@@ -0,0 +1,1898 @@
+# This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
+- id: create_tables
```

Comment:
In our daily performance test case, these inputs are `source`s or `table`s ? If they are `source`s, I think it is better to create sources instead of tables, since we found that the subqueries unseating optimization doesn't work for sources for a long time until we test it.

Reply:
I see, let me change it.